### ISSUE TITLE
Docs overhaul

### ISSUE THREAD

**Issue by Chris-Petty** [sussol/react-native-data-table#48]

**Comment by LouLomas** [comment]
@Chris-Petty still relevant?

**Comment by Chris-Petty** [comment]
Yes. There are bunch here relating to data table rewrite. We'll get to do it #soon

**Comment by MTayyabMehar** [comment]
please provide a document about "how to use it in our apps".

**Comment by MTayyabMehar** [comment]
There must be a document for react-native-data-table like all other packages, which should cover all aspects about how we can use and integrate it in our app.

**Comment by LouLomas** [comment]
@Chris-Petty can you please provide an update for @MTayyabMehar on how this is progressing?
cheers, Lou

**Comment by MakhouT** [comment]
Also needing this component, by any chance a timeframe @Chris-Petty ?

**Comment by Chris-Petty** [comment]
Hey @MakhouT, we're in the progress of rewriting at the moment, probably a few weeks away. API is going to change drastically to take advantage of React Native VirtualisedList. Docs rewrite will follow.
